wordpress-mobile/WordPress-iOS
# Integrated updates to both WordPressKit and WordPressAuthenticator

### Description
Propagates the fix for [this issue](
https://github.com/wordpress-mobile/WordPressKit-iOS/pull/80) in `WordPressKit` to WPiOS. Because this was an update to the major version, a corresponding update to `WordPressAuthenticator` was also required.

### Testing
Checkout the branch and confirm that existing tests pass.

To test:

- Checkout the branch and confirm that existing tests pass.
- Perform a smoke test of the app and confirm nothing looks off. You should be able to exercise network requests without issue.

Update release notes:
